packages/drivers/odsp-driver/src/index.ts

```diff
@@ -50,3 +50,5 @@ export {
 } from "./odspFluidFileLink";
 
 export { parseCompactSnapshotResponse } from "./compactSnapshotParser";
+
+export { SnapshotPrefetchResultCache } from "./odspCache";
```

Comment:
why do we need to export it from the driver package?

Reply:
Removed.